sql_jobs/hourly/skill_ranks.sql

```diff
@@ -0,0 +1,72 @@
+CREATE TABLE IF NOT EXISTS skill_ranks (
+    id bigint,
+    normalized_name varchar(128),
+    name varchar(128),
+    total_count numeric
+);
+
+CREATE TABLE skill_ranks_tmp
+AS SELECT * FROM skill_ranks;
```

Comment:
If this grows large in the future, there might be a speed improvement around limiting this select, as we're throwing out everything but the schema anyway.

Reply:
yeah -def makes sense.